yarn.lock

```diff
@@ -1766,11 +1766,6 @@
   dependencies:
     "@types/glob" "*"
 
-"@types/aws-lambda@^8.10.85":
```

Comment:
Are you expecting changes to yarn.lock?

Reply:
I'm not - do you know why this happens sometimes?

Reply:
It might be because you use `yarn install` without passing the arg `--frozen-lockfile`.

I do think this arg should be included in the CONTRIBUTING.md .

Reply:
Thanks @kirintwn!

@corymhall my experience is limited here, would we want to avoid having users update the lockfile?